<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game24Widget</class>
 <widget class="QWidget" name="Game24Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>24点游戏</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget { background-color: #0f172a; color: #f9fafb; }
QLabel  { font-size: 16px; }
QPushButton {
    background-color: #10b981;
    color: white;
    padding: 8px 16px;
    border-radius: 4px;
    border: none;
    font-size: 16px;
}
QPushButton:disabled { background-color: #6b7280; }
QPushButton:hover:!disabled { background-color: #059669; }
QLineEdit {
    background-color: #111827;
    border: 1px solid #374151;
    padding: 6px;
    border-radius: 4px;
    font-size: 18px;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="margin" stdset="0">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QLabel" name="m_userLabel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>玩家：未知</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_roundLabel">
       <property name="text">
        <string>第 1 局</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_timerLabel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>00:60</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_scoreLabel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>总分：0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="cardsLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="m_cardLabels_0">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #1e293b; border: 3px solid #475569; border-radius: 12px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_cardLabels_1">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #1e293b; border: 3px solid #475569; border-radius: 12px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_cardLabels_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #1e293b; border: 3px solid #475569; border-radius: 12px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_cardLabels_3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>140</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>140</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #1e293b; border: 3px solid #475569; border-radius: 12px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="exprLayout">
     <item>
      <widget class="QLabel" name="exprTipLabel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>请输入计算表达式（或输入 NO 放弃）：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_exprEdit">
       <property name="placeholderText">
        <string>例如：(3+5)*2+4</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btnLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_startBtn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>开始游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_submitBtn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>提交答案</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_nextBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>下一局</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
